src/engine/engine_vis_visualize.c

```diff
@@ -62,7 +62,7 @@ static void makeLabel(const mjModel* m, mjtObj type, int id, char* label) {
   }
 
   // copy result into label
-  strncpy(label, txt, 99);
+  strncpy(label, txt, 100);
```

Comment:
Use `mjSTRNCPY` and then you can delete the following line.

Reply:
Can't use `mjSTRNCPY` because the type of `label` is `char*` which would cause compiler error.
https://github.com/google-deepmind/mujoco/blob/1885d518ebf80b6fe6c2ba595be06027ce7306da/src/engine/engine_array_safety.h#L25-L27

Reply:
Right, sorry.